src/CISDSCResourceGeneration/functions/private/Import-AudicCsv.ps1

```diff
@@ -26,20 +25,16 @@ function Import-AudicCsv {
     }
 
     process {
-        [System.Collections.ArrayList]$ScaffoldingBlocks = @()
-
         Get-ChildItem -Path $GPOPath -Filter 'Audit.csv' -Recurse | ForEach-Object -Process {
             $CSV = Import-CSV -Path $_.FullName
 
             foreach($Row in $CSV){
                 switch ($Row){
-                    {!([string]::IsNullOrEmpty($Row.'Subcategory GUID'))} {$ScaffoldingBlocks += ConvertFrom-AuditPolicySubcategoryRawGPO -SubcategoryGUID $Row.'Subcategory GUID' -Subcategory $Row.Subcategory -InclusionSetting $Row.'Inclusion Setting'}
+                    {!([string]::IsNullOrEmpty($Row.'Subcategory GUID'))} {ConvertFrom-AuditPolicySubcategoryRawGPO -SubcategoryGUID $Row.'Subcategory GUID' -Subcategory $Row.Subcategory -InclusionSetting $Row.'Inclusion Setting'}
```

Comment:
Are we doing implicit return here on purpose? The pattern is hard on debuggers if we ever need to diagnose an issue here later, since there's no variable to grab in a watch-list.

Reply:
These cmdlets likely need renamed in a future update to make it more clear. They no longer return anything. They essentially do some data manipulation and store it in an existing object in script scope

Reply:
Previously these "scaffoldingblock" objects where created and then later matched to a recommendation now they are the same object.